modules/460-log-shipper/hooks/internal/vector/source/kubernetes.go

```diff
@@ -140,6 +146,9 @@ func NewKubernetes(name string, spec v1alpha1.KubernetesPodsSpec, namespaced boo
 			PodNodeName:    "node",
 			PodOwner:       "pod_owner",
 		},
+		nodeAnnotationFields: NodeAnnotationFields{
+			NodeLabels: "node_labels",
```

Comment:
Not all node labels should be exposed to logs, only the label with the node group.  The case with the missing label should also be covered.